node/db/src/rocks.rs

```diff
@@ -50,7 +59,8 @@ impl RocksDb {
             db_opts.set_compaction_style(compaction_style);
             db_opts.set_disable_auto_compactions(false);
         } else {
-            db_opts.set_disable_auto_compactions(true);
+            db_opts.set_disable_auto_compactions(false);
```

Comment:
If you enable compaction then the mode will still be LEVEL, as it is the default:
see: https://github.com/rust-rocksdb/rust-rocksdb/blob/v0.19.0/src/db_options.rs#L2082

Calling in both `if` and `else` branchs `set_disable_auto_compactions(false)` will make impossible to run forest without compactions.

Reply:
IOW this will make forest run always **with** LEVEL compactions.

Reply:
You are right! I should revert the change here